tests/skipped_tests_gpu.tbl

```diff
@@ -1504,6 +1504,8 @@ tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsDiri
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_0_{scale_shape=(), shape=(4, 3, 2)}::test_exponential
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_1_{scale_shape=(), shape=(3, 2)}::test_exponential
 tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsExponential_param_2_{scale_shape=(), shape=None}::test_exponential
+tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsGeometric_param_0_{p_shape=(), shape=(4, 3, 2)}::test_geometric
+tests/third_party/cupy/random_tests/test_distributions.py::TestDistributionsGeometric_param_1_{p_shape=(), shape=(3, 2)}::test_geometric
```

Comment:
Is this needed for checking you function logic?

Reply:
What is the reason? Provide some info, please

Reply:
Likely due to `types_map` in the old code that multiplies array by scalar https://github.com/IntelPython/dpnp/blob/85aad6be56c4555855da62f0b21c978902448d24/dpnp/dpnp_algo/dpnp_algo_mathematical.pyx#L279-L290